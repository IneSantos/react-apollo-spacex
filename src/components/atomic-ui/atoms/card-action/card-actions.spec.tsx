import React from "react";
import { render } from "@testing-library/react";
import CardAction from "./card-action";

describe("CardAction atom tests", () => {
  test(`CardAction component render`, () => {
    const { container } = render(<CardAction name="CARD ACTION" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
