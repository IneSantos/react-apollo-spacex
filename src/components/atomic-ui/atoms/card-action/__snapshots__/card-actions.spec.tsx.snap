// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardAction atom tests CardAction component render 1`] = `
<a
  class="card-action "
>
   
  CARD ACTION
   
</a>
`;
